core/src/main/java/org/geysermc/geyser/session/GeyserSession.java

```diff
@@ -581,6 +581,19 @@ public class GeyserSession implements GeyserConnection, GeyserCommandSource {
 
     private MinecraftProtocol protocol;
 
+    /**
+     * A set of elements to hide when the player is in spectator mode.
+     * Helps with tidying up the GUI; Java-style.
+     */
+    private static final Set<GuiElement> SPECTATOR_HUD = Set.of(
+            GuiElement.AIR_BUBBLES_BAR,
```

Comment:
I might suggest having this in GeyserCameraData. Also the javadoc is good, but the name itself still sounds like it is the HUD that is present, rather than not present. Maybe `SPECTATOR_HIDDEN ELEMENTS` - up to you :)